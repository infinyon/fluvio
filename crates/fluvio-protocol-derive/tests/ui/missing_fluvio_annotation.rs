use fluvio_protocol_derive::{Decoder, Encoder};

#[derive(Clone, Encoder, Decoder)]
pub enum SmartModuleInvocationWasm {
    Predefined(String),
    AdHoc(Vec<u8>),
}
