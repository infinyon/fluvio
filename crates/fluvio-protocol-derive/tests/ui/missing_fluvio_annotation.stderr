error: You must provide `fluvio` annotation when using `Encode`/`Decode` derive macro
 --> tests/ui/missing_fluvio_annotation.rs:5:5
  |
5 |     Predefined(String),
  |     ^^^^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/missing_fluvio_annotation.rs:7:2
  |
7 | }
  |  ^ consider adding a `main` function to `$DIR/tests/ui/missing_fluvio_annotation.rs`
